File: py/wtf_loat2.py
#!/usr/bin/env python3

# this is just sad

print("Hey Python, does (0.7 + 0.2) + 0.1 equal 0.7 + (0.2 + 0.1)?")
b = ((0.7 + 0.2) + 0.1) == (0.7 + (0.2 + 0.1))
print("Python says: ", b)
if not b:
    print("Womp womp womp")

File: c/wtf_loat2.c
#include <stdio.h>

int main (void) {
    printf("Hey C, does ((0.7 + 0.2) + 0.1) == (0.7 + (0.2 + 0.1))?\n");
    int b_ = ((0.7 + 0.2) + 0.1) == (0.7 + (0.2 + 0.1));
    printf("C says: %s\n", b_ ? "True" : "False");

    if (! b_) {
        printf ("womp womp\n");
    } else {
        printf ("You got it?! Good job\n");
    }

    return 0;
}

